app/services/open_library_books.rb

```diff
@@ -0,0 +1,16 @@
+class OpenLibraryBooks
```

Comment:
Suena medio redundante, pero solemos aclarar que esta clase es un service agregando al final del nombre de la clase la palabra service:
 ```suggestion
class OpenLibraryBooksService
```
(cambiemos el nombre del archivo tambien: desde `app/services/open_library_books.rb` hacia `app/services/open_library_books_service.rb`)

El motivo es porque por lo general tenemos un archivo que se llama BookController así que también lo usamos para los service: BookService